<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="choose_file">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>420</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="log_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>761</width>
      <height>371</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>250</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Timestamp</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Level</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Message</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="search_bar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="search_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search Bar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;-- BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as JSON</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="counter">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Level</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Counter</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
